package.json

```diff
@@ -24,8 +24,8 @@
   "dependencies": {
     "cosmiconfig": "^3.0.1",
     "eslint": "^4.5.0",
-    "find-up": "^2.1.0",
     "pify": "3.0.0",
+    "resolve-from": "4.0.0",
```

Comment:
Please do not se this module; you should be able to achieve this with `resolve`

Reply:
I maintain it; I didn't create it.

No, it doesn't fall back to the built in function, but that's an implementation detail.

Reply:
tomato potato. Jest actually comes with it's own `jest-resolve` package which seems to be a fork of substack/node-resolve.

They did not write docs about it at all, so i would refrain from using that anyhow.